src/action/common/configure_nix.rs

```diff
@@ -21,23 +21,36 @@ pub struct ConfigureNix {
 }
 
 impl ConfigureNix {
+    pub fn typetag() -> &'static str {
```

Comment:
Maybe inappropriately, my gut is itching to make this a newtype so that we don't put random strings in there, but the typetag's we've constructed. That said, I don't think it _really_ matters all that much. You know how I get with newtypes, though :laughing:

Reply:
Let's do it.